src/test/java/posters/pageobjects/pages/checkout/NewPaymentPage.java

```diff
@@ -34,9 +34,10 @@
 
     @Override
     @Step("ensure this is a new payment page")
-    public void isExpectedPage()
+    public NewPaymentPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.